src/settings/FinePolicy/utils/normalize.test.js

```diff
@@ -0,0 +1,106 @@
+import { isEqual } from 'lodash';
+
+import normalize from './normalize';
+
+const dataWithValuesForRemove = {
+  closingType: {
+    feeFine: 'interval',
+    loan: 'interval',
+    loanExceptions: ['interval'],
+  },
+  feeFine: {
+    duration: 1,
+    intervalId: 'Days',
+  },
+  loan: {
+    duration: 1,
+    intervalId: 'Minutes',
+  },
+  loanExceptions: [
+    {
+      duration: 1,
+      intervalId: 'Months',
+      paymentMethod: 'MODFEE-206-payment-methods',
+    }
+  ],
+  treatEnabled: true,
+  overdueFine: {
+    quantity: '0'
+  },
+  overdueRecallFine: {
+    quantity: {}
+  },
+};
+
+const expectedResult = {
+  closingType: {
+    feeFine: 'interval',
+    loan: 'interval',
+    loanExceptions: ['interval'],
+  },
+  feeFine: {
+    duration: 1,
+    intervalId: 'Days',
+  },
+  loan: {
+    duration: 1,
+    intervalId: 'Minutes',
+  },
+  loanExceptions: [
+    {
+      duration: 1,
+      intervalId: 'Months',
+      paymentMethod: 'MODFEE-206-payment-methods',
+    }
+  ],
+  treatEnabled: true,
+};
+
+const dataWithoutValuesForRemove = {
+  closingType: {
+    feeFine: 'interval',
+    loan: 'interval',
+    loanExceptions: ['interval'],
+  },
+  feeFine: {
+    duration: 1,
+    intervalId: 'Days',
+  },
+  loan: {
+    duration: 1,
+    intervalId: 'Minutes',
+  },
+  loanExceptions: [
+    {
+      duration: 1,
+      intervalId: 'Months',
+      paymentMethod: 'MODFEE-206-payment-methods',
+    }
+  ],
+  treatEnabled: true,
+  overdueFine: {
+    quantity: 'test'
+  },
+  overdueRecallFine: {
+    quantity: 'test'
+  },
+};
+
+describe('Normalize function', () => {
+  it('should remove data correctly', () => {
+    const result = normalize(dataWithValuesForRemove);
+    expect(isEqual(expectedResult, result)).toBeTruthy();
+  });
+
+  it('should not remove data', () => {
+    const result = normalize(dataWithoutValuesForRemove);
+    expect(isEqual(result, dataWithoutValuesForRemove)).toBeTruthy();
+  });
+});
+
```

Comment:
Remove multiple empty lines in the end, you need to leave single empty line in the end of every file.

Reply:
oops, have no idea how it was happend =) refactored